controllers/responses.js

```diff
@@ -3,42 +3,42 @@ const Responses = require('../models/Responses');
 const Reports = require('../models/Reports');
 const moment = require('moment');
 const { endOfDay, subDays, startOfDay } = require('date-fns');
-const searchReports = require('../helpers/searchReports');
-
-router.get('/', async (req, res) => {
-	try {
-		const responses = await Responses.find();
-		const message = 'The responses were found in the database.';
-		res.status(200).json({ message, responses });
-	} catch (error) {
-		res.status(500).json({
-			message:
-				'Sorry but something went wrong while retrieving the list of responses.'
-		});
-
-		throw new Error(error);
-	}
-});
+const { searchReports, searchReportsByUser } = require('../helpers/searchReports');
+
+// router.get('/', async (req, res) => {
```

Comment:
In writing the docs, I'd actually just put a comment here asking "do we need this?"